docs/api/js/zksync2-js/accounts.md

```diff
@@ -139,7 +139,7 @@ import { Wallet, Provider, types } from "zksync2-js";
 
 const unconnectedWallet = new Wallet(PRIVATE_KEY);
 
-const provider = Provider.getDefaultProvider(types.Network.Goerli);
+const provider = Provider.getDefaultProvider(types.Network.Sepolia);
```

Comment:
does it actually work? Was Sepolia added to types?

Reply:
Yes, `Sepolia` is here : https://github.com/zksync-sdk/zksync2-js/blob/bc6ce3cefb5f0787d64c0ced07c086462bd63b4e/src/types.ts#L24